applications/ConvectionDiffusionApplication/python_scripts/convection_diffusion_explicit_solver.py

```diff
@@ -0,0 +1,115 @@
+# Importing the Kratos Library
+import KratosMultiphysics
+
+# Import applications
+import KratosMultiphysics.ConvectionDiffusionApplication as ConvectionDiffusionApplication
+
+# Import base class file
+from KratosMultiphysics.ConvectionDiffusionApplication import (
+    convection_diffusion_base_solver,
+)
+
+
+def CreateSolver(model, custom_settings):
+    return ConvectionDiffusionExplicitSolver(model, custom_settings)
+
+
+class ConvectionDiffusionExplicitSolver(
+    convection_diffusion_base_solver.ConvectionDiffusionBaseSolver
+):
+    """
+    The explicit class for convection-diffusion solvers.
+    See convection_diffusion_base_solver.py for more information.
+    """
+
+    def __init__(self, model, custom_settings):
+        # Construct the base solver and validate the remaining settings in the base class
+        super(ConvectionDiffusionExplicitSolver, self).__init__(model, custom_settings)
+
+        # Overwrite the base solver minimum buffer size
+        self.min_buffer_size = 2
+
+        element_name = self.settings["element_replace_settings"][
+            "element_name"
+        ].GetString()
+        if self.settings["use_orthogonal_subscales"].GetBool() is True:
+            if element_name in (
+                "QSConvectionDiffusionExplicit",
+                "DConvectionDiffusionExplicit",
+            ):
+                self.main_model_part.ProcessInfo.SetValue(
+                    KratosMultiphysics.OSS_SWITCH, 1
+                )
+            else:
+                err_msg = self.__class__.__name__
+                err_msg += (
+                    "The selected element",
+                    element_name,
+                    "does not support OSS projection. Select QSConvectionDiffusionExplicit or DConvectionDiffusionExplicit instead.",
+                )
+                raise Exception(err_msg)
+        else:
+            if element_name in (
+                "QSConvectionDiffusionExplicit",
+                "DConvectionDiffusionExplicit",
+            ):
+                self.main_model_part.ProcessInfo.SetValue(
+                    KratosMultiphysics.OSS_SWITCH, 0
+                )
+            else:
+                err_msg = self.__class__.__name__
+                err_msg += (
+                    "The selected element",
+                    element_name,
+                    "does not support OSS projection. Select QSConvectionDiffusionExplicit or DConvectionDiffusionExplicit instead.",
+                )
+                raise Exception(err_msg)
+
+        KratosMultiphysics.Logger.PrintInfo(
+            self.__class__.__name__, "Construction finished"
+        )
```

Comment:
```suggestion
        KratosMultiphysics.Logger.PrintInfo(self.__class__.__name__, "Construction finished")
```
Please, increase your maximum characters per line. I think in this case it makes readability much more difficult.

Reply:
Is there a default?

Reply:
People says 80. But I think this is something that comes from ancient screens. I'd say "common sense" to ensure that going through the code is as comfortable as possible.